projects/Amlogic/bootloader/release

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Again this file is sourced and will never work standalone, but this looks to be a common error. I'd suggest removing it here, we can then fix the other instances in a separate PR.